packages/utils/src/lib/report-to-md.ts

```diff
@@ -87,14 +95,41 @@ function reportToCategoriesSection(report: ScoredReport): string {
     )} Score:  ${style(formatReportScore(category.score))}`;
     const categoryDocs = getDocsAndDescription(category);
 
-    const refs = category.refs.reduce((acc, ref) => {
-      if (ref.type === 'group') {
-        acc += groupRefItemToCategorySection(ref.slug, ref.plugin, plugins);
-      } else {
-        acc += auditRefItemToCategorySection(ref.slug, ref.plugin, plugins);
-      }
-      return acc;
-    }, '');
+    const auditsAndGroups = category.refs.reduce(
+      (
+        acc: {
+          audits: WeighedAuditReport[];
+          groups: EnrichedScoredAuditGroupWithAudits[];
+        },
+        ref: CategoryRef,
+      ) => {
+        if (ref.type === 'group') {
+          acc.groups = [
+            ...acc.groups,
+            getGroupWithAudits(ref.slug, ref.plugin, plugins),
+          ];
+        } else {
+          acc.audits = [...acc.audits, getAuditByRef(ref, plugins)];
+        }
+        return { ...acc };
+      },
+      { groups: [], audits: [] },
+    );
+
+    const audits = auditsAndGroups.audits
+      .sort(sortCategoryAudits)
+      .reduce((acc, audit) => {
+        acc += auditItemToCategorySection(audit);
+        acc += NEW_LINE;
+        return acc;
+      }, '');
```

Comment:
Doesn't need to be `reduce`, you can use `map` and `join`:

```suggestion
      .map(auditItemToCategorySection)
      .join(NEW_LINE);
```

Reply:
Done